### PR TITLE
Document installation using binary packages

### PR DESCRIPTION
**Pull request by divarvel**

### CODE REVIEW

File: commons/clever-tools/getting_started.md [hunk 1]
```diff
@@ -16,24 +16,60 @@ applications from the command line with Clever Tools.
...
+You need to add both files (`clever.exe` and `nodegit.node`) to your `PATH` to use the application.
+
+### GNU/Linux
+
+A pre-compiled version is available: [clever-tools-0.9.0_linux.tar.gz](https://clever-tools.cellar.services.clever-cloud.com/releases/0.9.0/clever-tools-0.9.0_linux.tar.gz).
+You need to add both files (`clever` and `nodegit.node`) to your `PATH` to use the application.
+
+#### Autocompletion
+
+Clever Tools comes with a comprehensive auto-completion system.
```
Comment (urcadox) [inline commons/clever-tools/getting_started.md]: Not a big deal but why is this here and not in the Mac OS section as well?

File: commons/clever-tools/getting_started.md [hunk 2]
```diff
@@ -16,24 +16,60 @@ applications from the command line with Clever Tools.
...
+
+    # for zsh
+    clever --zsh-autocomplete-script $(which clever) | sudo tee /usr/share/zsh/site-functions
+
+#### In case of `libcurl` version mismatch
+
+On some distributions (eg. archlinux), a `libcurl` version mismatch between the system default and the expected version can happen. If you see `Error: /usr/lib/libcurl.so.4: version 'CURL_OPENSSL_3' not found (required by /usr/bin/nodegit.node)` in the logs, you can add the following alias to force using a compatible `libcurl` version:
+
+    alias clever='LD_PRELOAD=libcurl.so.3 clever'
+
```
Comment (urcadox) [inline commons/clever-tools/getting_started.md]: One needs to install the libcurl compatibility version as well; on Archlinux: `pacman -S libcurl-compat`.

File: commons/clever-tools/getting_started.md [hunk 3]
```diff
@@ -16,24 +16,60 @@ applications from the command line with Clever Tools.
...
+### Manual installation (advanced)
+
+Clever Tools is built with [npm](https://npmjs.org) if you wish to install it from `npm`, you can run `npm install -g clever-tools`. You can also install it [from source](https://github.com/CleverCloud/clever-tools).
```
Comment (urcadox) [inline commons/clever-tools/getting_started.md]: s/ if/. If/

File: commons/clever-tools/getting_started.md (hunk 1 above)
Comment (divarvel) [reply commons/clever-tools/getting_started.md]: Autocompletion is installed by the brew package. I can mention it though
